File: tests/fixtures/java/PydanticValidation.java
class CreateOwnerForm {
    @NotNull
    @Size(min = 1, max = 30)
    private String firstName;

    @Min(0)
    @Max(150)
    private int age;

    @Pattern(regexp = "[A-Z]+")
    @NotEmpty
    private String code;

    @Digits(integer = 3, fraction = 2)
    private int price;
}

File: tests/fixtures/python/PydanticValidation.py
from __future__ import annotations

from pydantic import BaseModel, Field


class CreateOwnerForm(BaseModel):
    first_name: str = Field(..., min_length=1, max_length=30)
    age: int = Field(..., ge=0, le=150)
    code: str = Field(..., min_length=1, pattern='[A-Z]+')
    # @Digits(integer=3, fraction=2)
    price: int = Field(...)
